psi4/src/psi4/lib3index/cholesky.h

```diff
@@ -29,6 +29,7 @@
 #ifndef THREE_INDEX_CHOLESKY
 #define THREE_INDEX_CHOLESKY
 
+#include "psi4/psi4-dec.h"
```

Comment:
This is the header that's needed rather than the smaller `pragma.h`?

Reply:
+1 for this. Anything to keep headers simpler, especially in other headers is good.